src/TriggerUnit.cpp

```diff
@@ -448,7 +448,7 @@ void TriggerUnit::doCleanup()
 
 void TriggerUnit::markCleanup( TTrigger * pT )
 {
-    for(auto & trigger : mCleanupList)
+    for(auto trigger : mCleanupList)
```

Comment:
I think I commented elsewhere on this - if mCleanupList (a `QLIst<TTrigger *>` IIRC) was, instead a `QSet<TTrigger *>` we would not have to worry about whether it was _already_ in (and not add it if it was already) as `QSet<T>::insert(...)` automatically works as we want as a set can only contain one instance of each value...